### ISSUE TITLE
Error on opening a file

### ISSUE THREAD

**Issue by gurumaia** [h-hirokawa/atom-autocomplete-ansible#18]
I'm getting this error whenever I open a file, followed by the plugin not working:
```
Command failed: /usr/bin/python "/home/gustavo/.atom/packages/autocomplete-ansible/lib/parse_ansible.py"
Traceback (most recent call last):
  File "/home/gustavo/.atom/packages/autocomplete-ansible/lib/parse_ansible.py", line 11, in <module>
    from ansible.plugins import lookup_loader, module_loader
ImportError: cannot import name lookup_loader
```

Note that I'm using ansible 2.4-devel, running from source. If I revert back to 2.3 it works fine.

Thanks

**Comment by alexbarton** [comment]
I'm seeing this as well:
```
Command failed: /usr/local/Cellar/ansible/2.4.0.0/libexec/bin/python2.7 "/Users/alex/.atom/packages/autocomplete-ansible/lib/parse_ansible.py"
Traceback (most recent call last):
  File "/Users/alex/.atom/packages/autocomplete-ansible/lib/parse_ansible.py", line 11, in <module>
    from ansible.plugins import lookup_loader, module_loader
ImportError: cannot import name lookup_loader
```

I think this is related to ansible 2.4, which seems to be unsupported, see the note on the front page of atom-autocomplkete-ansible:

> Requirements:
>  ansible >= 2.0, < 2.4

So most probably the title of this issue should be changed to "**Please support Ansible 2.4**", no?

**Comment by h-hirokawa** [comment]
The issue is fixed in v0.1.15 (#23).

**Comment by alexbarton** [comment]
Thanks a lot for your work, @h-hirokawa!
